node/resolvers/Mutations/Users.ts

```diff
@@ -197,7 +195,7 @@ export const addUser = async (_: any, params: any, ctx: Context) => {
   } = ctx
 
   try {
-    const cId = await addUserToMasterdata({ masterdata, params })
+    const cId = await addUserToMasterdata({ masterdata, params, ctx })
```

Comment:
I just noticed a few lines below this one, there is a block checking if the user already exists in the organization/cost center... Why it was not working before? I think it doesn't make sense to keep both checks, we just need one of them...

(lets discuss offline)

Reply:
actually, this PR https://github.com/vtex-apps/b2b-organizations-graphql/pull/157 resolves everything, we can close this one.